Debug App/Tests/Unit Tests/Modules/HUC_TokenizationViewModelTests.swift

```diff
@@ -354,7 +352,7 @@ class HUC_TokenizationViewModelTests: XCTestCase {
             }
             
         } else {
-            self.paymentCompletion = { checkoutData, err in
+            self.paymentCompletion = { _, _ in
```

Comment:
AFAICT, these arguments can be removed entirely from the closure definition in the future